lib/msf/core/payload/malleable_c2.rb

```diff
@@ -0,0 +1,461 @@
+# -*- coding: binary -*-
+
+##
+# This module contains helper functions for parsing and loading malleable
+# C2 profiles into ruby objects.
+##
+
+require 'strscan'
+require 'rex/post/meterpreter/packet'
+
+# Handle escape sequences in the strings provided by the c2 profile
+class String
```

Comment:
It'd be great to pop this into the module itself instead of patching it globally onto the `String` class 🤞